src/codec/id.rs

```diff
@@ -907,7 +923,7 @@ impl From<AVCodecID> for Id {
 			AV_CODEC_ID_HDMV_TEXT_SUBTITLE => Id::HDMV_TEXT_SUBTITLE,
 
 			/* other specific kind of codecs (generally used for attachments) */
-			AV_CODEC_ID_TTF       => Id::TTF,
+			AVCodecID_AV_CODEC_ID_TTF       => Id::TTF,
```

Comment:
This is intentional. In ffmpeg that AVCodecID is defined like this: https://github.com/FFmpeg/FFmpeg/blob/release/3.3/libavcodec/avcodec.h#L667. As I understand rust cannot have enum parts with same values, so bindgen generated this binding like this:
```
pub const AVCodecID_AV_CODEC_ID_TTF: AVCodecID = AVCodecID::AV_CODEC_ID_FIRST_UNKNOWN;
```
This should get better, when rust stabilizes associated consts.